critical_demand.py

```diff
@@ -239,24 +241,27 @@
     label="electricity_demand",
     inputs={
         b_el_ac: solph.Flow(
-            fix=non_critical_demand, #/ non_critical_demand.max(),
-            #nominal_value=1, #non_critical_demand.max(),
+            min=(1-demand_reduction_factor)* (non_critical_demand/ non_critical_demand.max()),
+            max=(non_critical_demand/ non_critical_demand.max()),
```

Comment:
It is not clear to me why we also not multiply `max ` `1-demand_reduction_factor`

Reply:
if you can pick a value between min and max and min=max, then you don't choose a value, it has to be min, or max (because they are then same)

Reply:
(this would be equivalent to using fix)